src/SumoCoders/FrameworkCoreBundle/Tests/Controller/DatePickerControllerTest.php

```diff
@@ -148,44 +100,26 @@ public function testIfSingleTextWithDatePickerAndOnlyDatesBetweenCorrectly()
         $endDate = new \DateTime('next friday');
 
         $element = $crawler->filter('input#form_date_example8');
```

Comment:
don't forget to adjust this id when shifting the ids in the code

Reply:
fixed 89a0a2e